concourse/client.py

```diff
@@ -215,20 +231,42 @@ def _delete(self, url: str):
 
     @ensure_annotations
     def login(self, team: str, username: str, passwd: str):
```

Comment:
so this method would need to be implemented twice (ConcourseApiV3 + ConcourseApiV4)